packages/edit-site/src/components/dataviews/style.scss

```diff
@@ -34,9 +34,15 @@
 	}
 	th {
 		text-align: left;
-		font-weight: normal;
-		padding: 0 $grid-unit-20 $grid-unit-20;
-		color: $gray-700;
+		/*
+		 * This makes the Actions column header to use the same styles as the other ones.
+		 * The other column headers use the .components-button styles.
+		 */
+		&[data-field-id="actions"] {
```

Comment:
Should this just be `[data-field-id]`, or even something higher up? Mainly it seems like all items in this row should always be styled the same, even future additions.

Reply:
Yeah, good point. I think we can just target the `th`. The mismatch Joen describes can be observed by adding a field that is not hideable or sortable.

Reply:
Good one, thanks! Pushed [43190dc](https://github.com/WordPress/gutenberg/pull/56654/commits/43190dc213d3fb4c2f1081dab5d4b280b57fc75b)